front/src/common/Map/Layers/extensions/SNCF/NeutralSectionSigns.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { isNil } from 'lodash';
+import { useSelector } from 'react-redux';
+import { LayerProps, Source, SymbolLayer } from 'react-map-gl/maplibre';
+import { MAP_URL } from 'common/Map/const';
+import { Theme } from 'types';
+import OrderedLayer from 'common/Map/Layers/OrderedLayer';
+import { getMap } from 'reducers/map/selectors';
+import { LayerContext } from 'common/Map/Layers/types';
+import getKPLabelLayerProps from 'common/Map/Layers/KPLabel';
+import { getCommonSignsMastLayerProps } from 'common/Map/Layers/MastsLayers';
+
+export const neutralSectionSourceTable: string = 'neutral_signs';
+
+/**
+ * Base config of the neutral section signs layer.
+ * filter elements: Baisser Pantographe, Couper Courant, Reversible
+ * @param sourceTable
+ * @param prefix
+ */
+export function getNeutralSectionSignsLayerProps({
+  sourceTable,
+  prefix,
+}: Pick<LayerContext, 'sourceTable' | 'prefix'>): Omit<SymbolLayer, 'source'> {
+  const res: Omit<SymbolLayer, 'source'> = {
+    id: 'neutralSectionSignParams',
+    type: 'symbol',
+    minzoom: 11,
+    paint: {},
+    layout: {
+      'icon-image': ['concat', prefix, ['get', 'type']],
+      'icon-size': ['step', ['zoom'], 0.3, 15, 0.4],
+      'icon-offset': [
+        'step',
+        ['zoom'],
+        ['literal', [1.5, 0]],
+        13,
+        [
+          'case',
+          ['==', ['get', 'side'], 'RIGHT'],
+          ['literal', [55, -80]],
+          ['==', ['get', 'side'], 'LEFT'],
+          ['literal', [-55, -80]],
+          ['literal', [0, 0]],
+        ],
+      ],
+      'icon-rotation-alignment': 'map',
+      'icon-rotate': ['get', 'angle'],
+      'icon-allow-overlap': true,
+      'icon-ignore-placement': false,
+    },
+  };
+
+  if (!isNil(sourceTable)) res['source-layer'] = sourceTable;
+
+  return res;
+}
+
+type NeutralSectionSignsProps = {
+  infraID: number | undefined;
+  colors: Theme;
+  layerOrder?: number;
+};
+
+/**
+ * Renders the layer for the neutral sections signs
+ * https://osrd.fr/en/docs/explanation/models/neutral_sections
+ * @param props
+ * @constructor
```

Comment:
Nice to put this documentation, but it should be in the NeutralSection file, not the NeutralSectionSign one :) Can you move it there ? Also you can remove the param and constructor lines
```suggestion
 * Renders the layer for the neutral sections signs
 * https://osrd.fr/en/docs/explanation/models/neutral_sections
```